src/pages/productDetail/seulgi.js

```diff
@@ -0,0 +1,62 @@
+import './seulgi.scss';
+import { useState, useEffect } from 'react';
+
+const ProductDetail = () => {
+  const [list, setList] = useState([]);
+  const { itemDetail, itemName, description, price } = list[0];
+  useEffect(() => {
+    fetch('/Data/ITEM_LIST.json')
+      .then(res => res.json())
+      .then(function (result) {
+        return setList(result);
+      });
+  }, []);
```

Comment:
```suggestion
  const [list, setList] = useState({});
  const { itemDetail, itemName, description, price } = list;
  
  useEffect(() => {
    fetch('/Data/ITEM_LIST.json')
      .then(res => res.json())
      .then((result) => setList(result[0]));
  }, []);
```

- list라는 변수명은 어떤 것을 표현하고 있는지 파악하기 힘들어 보입니다!
- 다른 문맥들끼리는 공백 통해서 가독성 신경써주세요
- data 폴더는 소문자로 해주세요!!
  - 깃은 대소문자 추적을 기본적으로 하지 않아서 git mv 명령어를 통해서 대소문자 변경을 알려줘야 합니다.
  - https://dev.to/studentedge/rename-folder-to-lowercase-git-347d 참고해서 진행해보세요(이해 안되거나 어려우시면 라운지 시간에 멘토들에게 질문주세요!)
- 데이터가 배열형태로 오지만 그 중에서 첫번째 요소만 필요할 경우에는 애초에 setState 할 때 첫번째 아이템만 넣어줘도 될 것 같네요!
- 다른 함수의 인자로 들어가는 함수와 같은 경우에는 함수 선언식보다 arrow function 형태가 가독성이 더 좋습니다.

Reply:
멘토님 제가 브랜치 새로 생성 후 왔다갔다하면서 작업을 해서 productDetail 브랜치 부분도 같이 업로드 된 거 같습니다.
해당 부분은 productDetail 브랜치에서 수정해서 올리도록 할게요!

Reply:
해당 부분은 productDetail 브랜치에 수정하여 올렸습니다!